vaccine_feed_ingest/stages/enrichment.py

```diff
@@ -201,6 +221,47 @@ def _add_provider_tag(loc: location.NormalizedLocation) -> None:
     ]
 
 
+def _add_placekey_link(
+    loc: location.NormalizedLocation,
+    placekey_api: Optional[PlacekeyAPI],
+) -> None:
+    """Add placekey concordance to location if we can"""
+    if placekey_api is None:
+        return
+
+    if not loc.location:
+        return
+
+    if not _valid_address(loc):
+        return
```

Comment:
We call this here and in the function which calls this function. Can/should we optimize?

Reply:
I decided that answer is not to optimize because:
1. The check here is to make sure we can all placekey
2. The check later is to avoid importing locations that VIAL cannot handle.

In the future we want to remove check 2 to load locations with invalid addresses to VIAL, but we can't remove check 1.